src/features/account/hooks.ts

```diff
@@ -82,15 +82,15 @@ export function useAccountDetails(address?: Address, addressOrVoteSigner?: Addre
     query: { enabled: !!address },
   });
 
-  const isValidatorResult = useReadContract({
+  const isValidatorOrVoteSignerResult = useReadContract({
     address: Addresses.Validators,
     abi: validatorsABI,
     functionName: 'isValidator',
     args: [addressOrVoteSigner || ZERO_ADDRESS],
     query: { enabled: !!addressOrVoteSigner },
   });
 
-  const isValidatorGroupResult = useReadContract({
+  const isValidatorGroupOrVoteSignerResult = useReadContract({
```

Comment:
nit: slightly prefer the old name. voteSigners in general CAN Delegate just not those who sign for validators/groups. 

but i could have a locked gold contract that and add a signer that can vote o proposals.